packages/Card/stories/examples/DeluxeCard.js

```diff
@@ -14,7 +14,7 @@ export default () => {
           <Card.Title>Asset type name</Card.Title>
           <Input />
           <div style={{ width: "100%", height: "24px" }} />
-          <Checkbox>Use a display name</Checkbox>
+          <Checkbox onChange={() => {}}>Use a display name</Checkbox>

```

Comment:
onChange shouldn't be a required value, maybe you can remove it and add a default prop with that empty function. 

There are cases where the checkbox value is provided by an upper state and might not be handle by the user.